### PR TITLE
Ghd adl-002

### PR DESCRIPTION
**Pull request by cujomalainey**

### CODE REVIEW

**Comment by cujomalainey** [comment]
This patch switches the test hotword detector to the google one, let me know if that is a problem on this release branch and then I will include the patch to fix that

**Comment by sathya-nujella** [comment]
+ @sathyap-chrome , @Vamshigopal

**Comment by lgirdwood** [comment]
@mwasko @abonislawski fyi

**Comment by mwasko** [comment]
> This patch switches the test hotword detector to the google one, let me know if that is a problem on this release branch and then I will include the patch to fix that

That will probably be a blocker for Intel FW validation to run a full test scope on the releases branch.

File: src/audio/google_hotword_detect.c [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+
+struct comp_data {
+	struct comp_data_blob_handler *model_handler;
+	struct kpb_event_data event_data;
+	struct kpb_client client_data;
+
+	struct sof_ipc_comp_event event;
+	struct ipc_msg *msg;
+
+	int detected;
```
[the remark this answers is not in the document]
Comment (lgirdwood) [reply src/audio/google_hotword_detect.c:56]: 32bit logic is less instructions on xtensa, bytes require some extra for masking/shifting.

[the remark this answers is not in the document]
Comment (lyakh) [reply src/audio/google_hotword_detect.c:56]: @lgirdwood what about s8i, l8ui? Or do these instructions take more cycles themselves?

**Comment by cujomalainey** [comment]
@mwasko please approve so we can unblock releases for Chrome

**Pull request merged**